helm/examples/azure/README.md

```diff
@@ -1,4 +1,5 @@
 # HPCC Azure storage
+For Kubernetes server version 1.21+ the Azure Files Container Storage Interface (CSI) driver is used by Azure Kubernetes Service (AKS) to manage the lifecycle of Azure Files shares. Most AKS regions have Kubernetes version 1.21 and above now. In this examples/azure only no CSI driver storage is "Dynamically generate a storage account and shared volume with Storage Class" which has CSI provisioner as default by user can select legacy "kubernetes.io/azure-file" in values.yaml.
```

Comment:
>as default by user

by -> but ?

Reply:
>this examples/azure only no CSI driver storage is "Dynamically generate a storage account and shared volume with Storage Class"

I think may need rewording, I'm not sure what it's saying.

Reply:
perhaps: 

For Kubernetes server version 1.21+ the Azure Files Container Storage Interface (CSI) driver is used by Azure Kubernetes Service (AKS) to manage the lifecycle of Azure File shares. Currently most AKS regions are using Kubernetes version 1.21 and above. In the examples, Azure only no CSI driver storage will "Dynamically generate a storage account and shared volume with Storage Class" which will have a CSI provisioner as default. The user can select to use the legacy "[kubernetes.io/azure-file](http://kubernetes.io/azure-file)" in the values.yaml.

Reply:
fixed